example/presto_cluster/main.tf

```diff
@@ -1,28 +1,45 @@
 locals {
   nomad_datacenters = ["dc1"]
   nomad_namespace   = "default"
+
+  presto_vault_secret = {
+    use_vault_provider        = true
+    vault_kv_policy_name      = "kv-secret"
+    vault_kv_path             = "secret/data/dev/presto"
+    vault_kv_secret_key_name  = "cluster_shared_secret"
+  }
+  minio_vault_secret = {
+    use_vault_provider        = true
+    vault_kv_policy_name      = "kv-secret"
+    vault_kv_path             = "secret/data/dev/minio"
+    vault_kv_access_key_name  = "access_key"
+    vault_kv_secret_key_name  = "secret_key"
+  }
+  postgres_vault_secret = {
+    use_vault_provider     = true
+    vault_kv_policy_name   = "kv-secret"
+    vault_kv_path          = "secret/data/dev/postgres"
+    vault_kv_username_name = "username"
+    vault_kv_password_name = "password"
+  }
```

Comment:
Before I suggest removing them, is there any reason you added these in `locals` instead of writing them explicitly where applicable?

Reply:
Yeah, I will remove them. I will do it explicit